relay-general/src/protocol/contexts/device.rs

```diff
@@ -169,9 +172,80 @@ impl DeviceContext {
     }
 }
 
-#[test]
-fn test_device_context_roundtrip() {
-    let json = r#"{
+impl FromUserAgentInfo for DeviceContext {
+    fn from_client_hints(client_hints: &ClientHints) -> Option<Self> {
+        let device = client_hints.sec_ch_ua_model?.to_owned();
+        Some(Self {
+            family: Annotated::new(device),
```

Comment:
I know we talked about this, but I feel `sec-ch-ua-model` should go into the `model` field instead. Reading

https://github.com/getsentry/relay/blob/06c9d64bbaf9285274b690eaeed4992b97ce7213/relay-general/src/protocol/contexts/device.rs#L20-L23

and 

> A string containing the device version. For example "Pixel 3".

(from [spec](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Sec-CH-UA-Model))

Reply:
@jan-auer any opinion on this?